### ISSUE TITLE
flags are broken in MATE >= 1.6

### ISSUE THREAD

**Issue by infirit** [mate-desktop/libmatekbd#18]
No flag image is shown except an error icon. Confirmed on Mint 17 and gentoo.

```
gsettings set org.mate.peripherals-keyboard-xkb.indicator show-flags true
```

![keyboard_flag_error](https://cloud.githubusercontent.com/assets/3465730/3383060/04eeb2e2-fc45-11e3-9ee2-c88e82f66af6.png)
##

**Comment by monsta** [comment]
I also can confirm this. Happens in Debian Jessie with MATE 1.8.

**Comment by monsta** [comment]
Out of curiosity, I've installed an old release - Linux Mint 14 (which has MATE **1.4**) and... surprise, no flags there too. I've tried all the icon themes from the list that you can see on the screenshot.

![mint-14-no-flag-icons](https://cloud.githubusercontent.com/assets/5138986/8697400/64f09c18-2afd-11e5-8ff1-a648f93174c8.png)

Previously I thought that the feature had been broken during migration to GSettings, but apparently, it's not so.
The code simply tries to load an icon (using ```gtk_icon_theme_lookup_icon```) with the same name as the layout - in my case, it's either ```en``` or ```ru```. It fails because there's no such icon in the theme.

**Comment by vkotovv** [comment]
Still seeing this issue on Ubuntu 16.04 with mate-desktop 1.18 installed. Workaround from #11 helps though.

**Comment by raveit65** [comment]
I added flags to mate-icon theme and released 1.16.1 and 1.18.1.
https://github.com/mate-desktop/mate-icon-theme/commit/9f6615a50d3e5482dcaa56fbc44d9a57047ca49e
Flags are displayed out of box now in gtk3 and gt2 versions of MATE.
As always tested on fedora-26/25/24.
